data/en/isdateobject.json

```diff
@@ -6,7 +6,7 @@
 	"related":["isDate","isObject"],
 	"description":"Checks if the given value is a date / time object. The difference between isDateObject and isDate is that the latter returns true for date / time objects as well as date / time strings.",
 	"params": [
-		{"name":"dateObject","description":"A value, usually the name of a variable","required":true,"default":"","type":"any","values":[]}
+		{"name":"value","description":"A value, usually the name of a variable","required":true,"default":"","type":"any","values":[]}
```

Comment:
I still say get rid of the "usually the name of a variable" part. It's misleading.

Reply:
So, just say "A value" in the description?

Reply:
Up to you how you want to word it. You could say something like "A value to test whether it's a date object".